src/plasma/plasma_manager.c

```diff
@@ -1376,9 +1376,10 @@ void process_message(event_loop *loop,
   case MessageType_PlasmaFetchRequest: {
     LOG_DEBUG("Processing fetch remote");
     int64_t num_objects = plasma_read_FetchRequest_num_objects(data);
-    object_id object_ids_to_fetch[num_objects];
+    object_id *object_ids_to_fetch = malloc(num_objects * sizeof(object_id));
     plasma_read_FetchRequest(data, object_ids_to_fetch, num_objects);
     process_fetch_requests(conn, num_objects, &object_ids_to_fetch[0]);
+    free(object_ids_to_fetch);
```

Comment:
There is only one malloc per loop, because only one operation is done per loop. So not sure if reusing makes sense here. If the malloc becomes a bottleneck, we can create a dynamically resizable array that is reused across process_message iterations, but it makes the code less clear so I'd like to avoid doing it unless we know it is a bottleneck.